pkg/common/config/config.go

```diff
@@ -62,6 +62,10 @@ const (
 	// interval after which successful CnsRegisterVolumes will be cleaned up.
 	// Current default value is set to 12 hours
 	DefaultCnsRegisterVolumesCleanupIntervalInMin = 720
+	// DefaultVolumeMigrationCRCleanupIntervalInHours is the default time
+	// interval after which stale CnsVSphereVolumeMigration CRs will be cleaned up.
+	// Current default value is set to 24 hours
+	DefaultVolumeMigrationCRCleanupIntervalInHours = 24
```

Comment:
DefaultVolumeMigrationCRCleanupIntervalInHours => DefaultVolumeMigrationCRCleanupIntervalInMinutes
24 => 120

Reply:
Setting this to 60